chore(deps)!: fix new mongoose version

adriasala124 opened the pull request:
BREAKING CHANGE: fix new mongoose version

Force major version bump

Ref. #133

Commit:
chore(deps)!: fix new mongoose version

BREAKING CHANGE: fix new mongoose version

Commit:
chore(deps)!: fix mongoose new version

BREAKING CHANGE: fix mongoose new version

pierissimo reviewed (commented):
It's usually better to leave range versions for the peerDependencies.
What's the reasoning behind this PR?

Commit:
chore(deps): update package-locks to avoid skipLibCheck

adriasala124 commented:
> It's usually better to leave range versions for the peerDependencies. What's the reasoning behind this PR?

Hey @pierissimo! This PR is to generate a major release, which was not correctly triggered in the last PR.
Updating to latest mongoose 6.6.4 (instead of 6.6.2, previous PR) it was giving type errors, even with the skipLibCheck option. That's why I removed the ^.
But updating the package-locks is working fine even without the skipLibCheck, so I removed it and put back the ^

Commit:
Revert "chore(deps): update package-locks to avoid skipLibCheck"

This reverts commit 1a1ec607e3875c4a149d63ba30d252c86cd44202.

Commit:
chore(deps): redo package-locks to fix build dependency problems
